src/components/formElements/ButtonIcon.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <button
+    class="button-icon"
+    @click="$emit('click')"
+  ><img
+    class="icon"
+    :src="icon" width="25px"
```

Comment:
and why the width is not in css or at least in props with default '25px' for example? What do you think about that?